src/components/Details/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Details extends React.Component {
+    goToSearchPage = () => {
+        this.props.goToSearchPage();
+    }
+
+    render() {
+        const {item} = this.props;
+        const {poster_path, title, vote_average, tagline, release_date, runtime, overview} = item;
+
+        return (
+            <div className="details_wrapper">
+                <button 
+                    className="details_search_button"
+                    onClick={this.goToSearchPage}>search
```

Comment:
Is it needed to create a function `goToSearchPage ` in the component?
 Why not just ` onClick={this.**props**.goToSearchPage}>search`